src/rewards/interfaces/reward-schedules-vault.interface.ts

```diff
@@ -0,0 +1,19 @@
+import { VaultDefinition } from '../../vaults/interfaces/vault-definition.interface';
+
+export interface EmissionSchedule {
```

Comment:
this will be exported by  the sdk, we can define here an extention of the interface preferably in its own interface file

Reply:
k